Octokit.Tests.Integration/Reactive/ObservableIssueCommentsClientTests.cs

```diff
@@ -1,126 +0,0 @@
-﻿using System;
```

Comment:
I've removed this file because it is fully similar to non-reactive implementation. and usually we have integration tests for non-reactive clients, because reactive ones use non-reactive clients internally.If it not sounds reasonable I going to revert it back :+1: